tests/MongoDBDriverReadPreference_getMaxStalenessMS-001.phpt

```diff
@@ -1,22 +1,22 @@
 --TEST--
-MongoDB\Driver\ReadPreference::getMaxStalenessMS()
+MongoDB\Driver\ReadPreference::getMaxStalenessSeconds()
 --FILE--
 <?php
 require_once __DIR__ . '/utils.inc';
 
 $tests = [
-    0,
+    90,
```

Comment:
I had to change this from 0 to 90 (probably, 60 would have been ok to for **now**), as otherwise libmongoc would return a "invalid read preference" error, albeit without saying *why*. Should we file that as a libmongoc bug as well?